lib/Optimizer/Transforms/CMakeLists.txt

```diff
@@ -50,6 +50,8 @@ add_cudaq_library(OptTransforms
   RefToVeqAlloc.cpp
   RegToMem.cpp
   StatePreparation.cpp
+  UnitarySynthesis.cpp
+  Utils.cpp
```

Comment:
No. Let's not have a "Utils" module. It ends up being a miscellaneous dumping ground of poorly organized code over time.